sdk/storage/azblob/README.md

```diff
@@ -0,0 +1,224 @@
+  ```
+* Import SDK in your code:
+  ```golang
+  import "github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+  ```
+
+## Code Samples
+
+`azblob` allows you to interact with three types of resources
+
+* The storage accounts.
+* The containers within those storage accounts.
+* The blobs (block blob/ page blob/ append blob) within those containers.
+
+To interact with these resources, start by creating the instances of each type
+
+### Types of credentials
+
+The clients support different forms of authentication. Cosmos accounts can use a Shared Key Credential, Connection
+String, or an Shared Access Signature Token for authentication. Storage account can use the same credentials as a Cosmos
+account and can use the credentials
+in [`azidentity`](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity)
+like `azidentity.NewDefaultAzureCredential()`.
+
+The azblob package supports any of the types that implement the `azcore.TokenCredential` interface, authorization via a
+Connection String, or authorization with a Shared Access Signature Token.
+
+#### 1. Creating the client from a shared key
+
+To use an account [shared key][azure_shared_key] (aka account key or access key), provide the key as a string. This can
+be found in your storage account in the [Azure Portal][azure_portal_account_url] under the "Access Keys" section or by
+running the following Azure CLI command:
+
+```bash
+az storage account keys list -g MyResourceGroup -n MyStorageAccount
+```
+
+Use Azure Active Directory (AAD) authentication as the credential parameter to authenticate the client:
+
+```golang
+cred, err := azidentity.NewDefaultAzureCredential(nil)
+handle(err)
+serviceClient, err := azblob.NewServiceClient("https://<myAccountName>.blob.core.windows.net/", cred, nil)
+handle(err)
+```
+
+##### 2. Creating the client from a connection string
+
+Depending on your use case and authorization method, you may prefer to initialize a client instance with a connection
+string instead of providing the account URL and credential separately. To do this, pass the connection string to the
+client's `NewServiceClientFromConnectionString` method. The connection string can be found in your storage account in
+the [Azure Portal][azure_portal_account_url] under the "Access Keys" section or with the following Azure CLI command:
+
+```bash
+az storage account show-connection-string -g MyResourceGroup -n MyStorageAccount
+```
+
+```golang
+connStr := "DefaultEndpointsProtocol=https;AccountName=<myAccountName>;AccountKey=<myAccountKey>;EndpointSuffix=core.windows.net"
+serviceClient, err := azblob.NewServiceClientFromConnectionString(connStr, nil)
+```
+
+##### Creating the client from a SAS token
+
+To use a [shared access signature (SAS) token][azure_sas_token], provide the token as a string. If your account URL
+includes the SAS token, omit the credential parameter. You can generate a SAS token from the Azure Portal
```

Comment:
Don't omit the credential parameter, use the `azcore.AnonymousCredential` instead